gapic-generator/templates/default/service/test/smoke.erb

```diff
@@ -1,10 +1,5 @@
 <%- assert_locals gem -%>
-# frozen_string_literal: true
-
-<%= render partial: "shared/license" %>
-
-# Auto-generated by gapic-generator-ruby v<%= Gapic::Generator::VERSION %> on <%= Time.now.strftime "%Y-%m-%d" %>.
```

Comment:
Any opinions on if we should keep the timestamp in the `_warning.erb`. My personal preference is no, as you have it now, but I assume this was put here for a reason at some point?

Reply:
I removed it because it breaks the expected output comparison. If we put it back then we will need to reconsider how we test the generator's output.

Reply:
Ok, I'm good without it (code will be tracked in vcs anyway) so let's stick with that.